scripts/scripts.js

```diff
@@ -1,21 +1,21 @@
 var start = 0;
 var stateCode = "states";
-var message = "Detailed View"
-var filter = "filter"
-var parksArray = []
-var slideIndex = {}
+var message = "Detailed View";
+var filter = "filter";
+var parksArray = [];
+var slideIndex = {};
 
 function previousPage() {
   if (start === 0) {
     alert("You are on the first page");
   } else {
-    start = start - 50;
+    start = start - 25;
```

Comment:
Is this for making it load quicker?

Reply:
Yes, and to reduce the memory the page uses. This is not your fault, but the images are not compressed or resized at all, and they slow the page down. To the user, 25 vs 50 isn't a huge difference in terms of easy browsing, but it does make a difference in the performance of the page.

Reply:
Got it. Thank you!